docs/Contributors_Guide/add_use_case.rst

```diff
@@ -207,58 +207,235 @@ Add Sphinx Documentation File
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
 In the corresponding documentation MET tool name directory
-(**docs**/*use_cases/met_tool_wrapper/<MET TOOL NAME>*) for a met_tool_wrappers
+(*docs*/*use_cases/met_tool_wrapper/<MET TOOL NAME>*) for a met_tool_wrappers
 use case OR category directory for a model_applications use case
-(**docs**/*use_cases/model_applications/<CATEGORY>*), add:
+(*docs*/*use_cases/model_applications/<CATEGORY>*), users will need to a add 
+Python Sphinx Documentation (.py) file with the same name as the METplus configuration file.
+
+The following is a discussion of the use case template and all of its sections.
+A link to an example template is available within the `METplus repository. <https://github.com/dtcenter/METplus>`_
+The example template should be used by users as a starting point, but will need to be completely 
+updated with the use case's information. The template applies to both met_tool_wrappers and model_applications use cases.
+When completing the template, users should read through each section and its description 
+below before filling in a section, as some sections may not apply 
+to the use case being documented. For real examples, users are encouraged to review
+existing `Model Applications <https://metplus.readthedocs.io/en/latest/generated/model_applications/index.html>`_
+use case documentation. Except for the Header and Path to Use Case Configuration File section,
+all lines should begin with the '#' character to signify text, followed by at least one space before
+the text content. These are already provided in the example template.
+
+* Use Case Template Description:
+
+  * Header and Path to Use Case Configuration File
+
+    * This section begins with {PrimaryStatAnalysisToolName}: Brief (12 words or less) 
```

Comment:
```suggestion
    * This section begins with {MET_Tool_Name}: Brief (12 words or less) 
```

Reply:
It is possible that the use case does not call a MET tool, e.g. UserScript that calls a script to run METcalcpy or METplotpy. `PrimaryStatAnalysisToolName` may be confusing because there is a tool called `StatAnalysis`. Maybe `PrimaryAnalysisToolName`?

Reply:
The problem with this change is sometimes people will list ALL of the tools they use, or list a tool like PlotDataPlane, which does no statistical verification, as the lead name. PrimaryStatAnalysisToolName is something Tara insisted to use as it conveys where the primary focus of the use case is.

Reply:
Great! Thanks for the discussion all. I like `PrimaryAnalysisToolName`. Appreciate it!